lib/hammer_cli/output/fields.rb

```diff
@@ -4,14 +4,16 @@ module Fields
 
   class Field
 
-    attr_reader :label
-    attr_reader :path
+    attr_accessor :label
```

Comment:
Do we want to enable the writers and allow change these form outside?

Reply:
Since we're [using labels](https://github.com/theforeman/hammer-cli/pull/275/files#diff-5429b05e7a5cdcd85f448cc1b33defe9R16) to set field's ID, which is used to address a field, I don't think it's a good idea now :)

But again, is [this](https://github.com/theforeman/hammer-cli/pull/275/files#diff-5429b05e7a5cdcd85f448cc1b33defe9R16) an acceptable way? We could allow label changing from outside, but then we shouldn't identify fields by labels I guess..

Reply:
Regarding setting the default for id I think it is okay. Maybe with 'key' preferred over the 'label'.
In the original comment I was referring more to all three attr_accessors in general. It does not feel correct to modify any of these attributes on existing field.  Wouldn't it be better to replace filed with a new field?

I can imagine changing label in container or collection could be awkward, so ability to change label might be helpful in some cases. The :id would need to be a method getting the id dynamically.